src/modules/AieraChat/components/Messages/MessageFactory/Block/Text/markdown.tsx

```diff
@@ -150,14 +171,42 @@ const CustomTableBody = ({ children }: CustomComponentProps) => (
     <tbody className="divide-y divide-gray-200 dark:divide-gray-700">{children}</tbody>
 );
 
-const CustomTableCell = ({ children }: CustomComponentProps) => <td className="px-3 py-2 text-sm">{children}</td>;
+const CustomTableCell = ({ children }: CustomComponentProps) => (
+    <td className="px-3 py-2 text-sm">
+        <SearchableWrapper>{children}</SearchableWrapper>
+    </td>
+);
 
 const CustomTableHeaderCell = ({ children }: CustomComponentProps) => (
     <th className="px-3 py-2 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider">
-        {children}
+        <SearchableWrapper>{children}</SearchableWrapper>
     </th>
 );
 
+// Custom components for text content
+const CustomParagraph = ({ children, ...props }: CustomComponentProps) => (
```

Comment:
This is the main reason why this PR is a draft - I'm branching off this branch to see if I can make this work using the pattern in [aiera-dashboard](https://github.com/aiera-inc/aiera-dashboard/blob/master/src/app/pages/App/Inbox/MessageDetail/styles.css), but I'm unsure how the `CustomCitation` is going to work with that.

Reply:
I think you can do both right? use css styling for some, but not the citations?